experimental/packages/exporter-trace-otlp-grpc/test/traceHelper.ts

```diff
@@ -97,7 +98,8 @@ export const mockedReadableSpan: ReadableSpan = {
     version: 1,
     cost: 112.12,
   })),
-  instrumentationLibrary: { name: 'default', version: '0.0.1' },
+  instrumentationLibrary: scope,
+  instrumentationScope: scope,
```

Comment:
I'm not sure the benefits of this but it creates a burden for SDK consumers -- they'll have to duplicate the field too in their test facilities.

Reply:
I left the alias to `instrumentationLibrary` on `ReadableSpan` for the benefit of users who might be using it in a `SpanProcessor`. This at least follows what the Ruby SDK does: https://github.com/open-telemetry/opentelemetry-ruby/blob/4b26b25785a11a22b609b43331e45be89e5e3dab/sdk/lib/opentelemetry/sdk/trace/span_data.rb#L52-L57. 

If we can remove it completely, I'd be in support, as it would clean up tests, etc.

Reply:
We can't remove this now since its breaking change, actually i don't see any other way right now